Update harness-platform-faqs.md (#5860)
harness/developer-hub: kb/platform/harness-platform-faqs.md

#### Does the docker delegate also show expiry message in UI if the image is older than expiry threshold ?

Yes, the docker delegates also will show expiry status in the UI if the image is beyond expiry threshold.

#### I'm using Entra ID (formerly Azure Active Directory) as my identity provider and some users aren't being assigned to their groups on login. Why is this happening?

> Due to a limitation in Entra ID, users with more than 150 groups may not be assigned any group, which could cause issues for members of >150 groups.  For more information, go to [Users in over 150 groups](/docs/platform/authentication/single-sign-on-saml/#users-in-over-150-groups).

Reviewer: ```suggestion
Due to a limitation in Entra ID, users with more than 150 groups might not be assigned any group, which could cause issues for members of >150 groups.  For more information, go to [Users in over 150 groups](/docs/platform/authentication/single-sign-on-saml/#users-in-over-150-groups).
```

> Due to a limitation in Entra ID, users with more than 150 groups might not be assigned any group, which could cause issues for members of >150 groups.  For more information, go to [Users in over 150 groups](/docs/platform/authentication/single-sign-on-saml/#users-in-over-150-groups).

Reviewer: ```suggestion
Due to a limitation in Entra ID, users with more than 150 groups might not be assigned any group, which could also cause issues for members of >150 groups.  For more information, go to [Users in over 150 groups](/docs/platform/authentication/single-sign-on-saml/#users-in-over-150-groups).
```

#### Does the docker delegate also show expiry message in UI if the image is older than expiry threshold ?

Yes, the docker delegates also will show expiry status in the UI if the image is beyond expiry threshold.

> #### I am using EntraID(formerly AAD) as my identity provider and I have just one user, or a small select set of users that are not getting assigned to their groups properly on login, is there any reason for this, and how would I tell?

Reviewer: ```suggestion
#### I'm using Entra ID (formerly Azure Active Directory) as my identity provider and some users aren't being assigned to their groups on login. Why is this happening?
```

> Commonly, users who are members of >150 groups will experience this issue, no group assignment at all because of a limitation in Azure AD(EntraID) where users with more than 150 groups require additional steps seen here:

Reviewer: ```suggestion
Due to a limitation in Entra ID, users with more than 150 groups may not be assigned any group, which could cause issues for members of >150 groups.  For more information, go to [Users in over 150 groups](/docs/platform/authentication/single-sign-on-saml/#users-in-over-150-groups).
```

> https://developer.harness.io/docs/platform/authentication/single-sign-on-saml/#users-in-over-150-groups

Reviewer: ```suggestion
```

> You can determine whether or not this is the problem, by using SAML tracer, or capturing the SAML assertion on user sign-in and look to see if an attribute 'groups.link' exists, if so this is the issue confirmed.

Reviewer: ```suggestion
You can check if this is the issue by using SAML tracer or capturing the SAML assertion during user sign-in. Look for an attribute called `groups.link`. If it exists, then the issue is confirmed.
```
